clients.json

```diff
@@ -1192,7 +1192,15 @@
     "description": "A Redis client in pure Octave",
     "authors": ["markuman"]
   },
-
+  
+  {
+    "name": "MatlabRedis",
+    "language": "Matlab",
+    "repository": "https://github.com/GummyJum/MatlabRedis",
+    "description": "Pure Matlab Redis interface for Matlab>=2014B"
+    "active": true
```

Comment:
Missing comma between these lines. That's why the check fail I think.

Reply:
Fixed. Thanks